examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>I would separate sampling and plots in different cells and split the explanation above so they are closer to the relevant plot. </p><p> </p><p>Maybe even add extra links or references, i.e. to the posterior predictive notebook when using plot_ppc. This probably depends on what is assumed as known to readers of this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>